cfy_manager/components/mgmtworker/mgmtworker.py

```diff
@@ -137,22 +131,11 @@ def _prepare_snapshot_permissions(self):
         common.sudo(['chgrp', const.CLOUDIFY_GROUP, '/opt/manager'])
         common.sudo(['chmod', 'g+rw', '/opt/manager'])
 
-    def _verify_mgmtworker_alive(self):
-        systemd.verify_alive(MGMTWORKER)
-
-    def _configure(self):
-        if is_premium_installed():
-            cluster = Cluster(skip_installation=False)
-            cluster.configure()
```

Comment:
Why is this not needed any more? Does the mgmtworker pick up settings on startup?

Reply:
sure, but the `Cluster` did nothing anyway. It did check that the restservice is up. Which it must be if we got to this place. It didn't configure anything.

So, I've just removed the whole damn file instead.

Currently, cluster-join stuff happens in restservice.py, in `_join_cluster` and `_run_syncthing_configuration_script` instead.